### PR TITLE
Fix text not rendering when rendering to a rendertexture

### PR DESCRIPTION
**Pull request by themuflon**

### CODE REVIEW

File: loom/vendor/nanovg/src/nanovg_lm_gl.h [hunk 1]
```diff
@@ -1093,7 +1093,17 @@ static void glnvg__triangles(GLNVGcontext* gl, GLNVGcall* call)
...
+    if (GFX::Graphics::getFlags() & GFX::Graphics::FLAG_INVERTED)
+    {
+        LGL->glDisable(GL_CULL_FACE);
+    }
+
     LGL->glDrawArrays(GL_TRIANGLES, call->triangleOffset, call->triangleCount);
+
+    if (GFX::Graphics::getFlags() & GFX::Graphics::FLAG_INVERTED)
+    {
+        LGL->glDisable(GL_CULL_FACE);
```
Comment (SmilyOrg) [inline loom/vendor/nanovg/src/nanovg_lm_gl.h]: Hmm.. why is it disabled twice?

Comment (themuflon) [inline loom/vendor/nanovg/src/nanovg_lm_gl.h]: Well, it shouldn't be.

**Comment by SmilyOrg** [comment]
Does this play ok with shapes with holes in them (we need a test for that in the example)? E.g. if you create a square with `lineTo` and then cut out a hole in it by creating a smaller square inside of it, but with the opposite winding.

**Comment by themuflon** [comment]
It's a separate rendering path used only for text quads.

**Comment by SmilyOrg** [comment]
Ah, seems cool then.

**Pull request merged**